### PR TITLE
Added the navbar and the home page

### PR DESCRIPTION
**Pull request by jfoyarzo**
The first stage of the "My Portfolio" project. I added the main files with their contents to be correctly displayed on mobile devices and made the changes requested in the first review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400;500;700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./assets/css/style.css">
+</head>
+
+<body>
+    <header>
+        <nav class="nav-bar">
+            <img src="./assets/img/image_geometry_1.png" alt="geometric shapes">
```
Comment (mgmediaweb) [inline index.html]: You should kindly use the images that make up the layout, such as background-image.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <main>
+        <div class="headline">
+            <div class="text-wrapper">
+                <h1>Felipe<br>Oyarzo</h1>
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of
+                    my works. If you like what you see and have a project you need coded, don't hesitate and contact me.
+                </p>
+            </div>
+            <button class="button" type="button" id="btnHeadline">Start collaboration</button>
+            <img src="./assets/img/image_geometry_2.png" alt="Geometric shapes" class="geometric">
```
Comment (mgmediaweb) [inline index.html]: You should kindly use the images that make up the layout, such as background-image.

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+.headline {
+  display: flex;
+  flex-direction: column;
+}
+
+.text-wrapper {
+  padding: 0 5%;
+  margin-bottom: 5%;
+}
```
Comment (mgmediaweb) [inline assets/css/style.css]: Kindly, you should use flexbox to center align the text.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @jfoyarzo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (3)

## Highlights 💯 
- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️

- Kindly, you can create a more descriptive pull request, it would be very PRO 🔥 .
- Kindly, you should follow the Figma design a bit better, this button text is too big.
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- You should kindly use semantic SECTION tags for each section of your page instead of using DIV.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jfoyarzo) [reply index.html]: Hello @mgmediaweb , thanks for your feedback :) 
The geometry images should be used as background images?

File: assets/css/style.css (hunk 3 above)
Comment (jfoyarzo) [reply assets/css/style.css]: Oh, ok. So this text-wrapper should have a display: flex property and I should justify the content to the center?
Thank you very much for your help.

File: index.html (hunk 1 above)
Comment (mgmediaweb) [reply index.html]: yes

File: index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        <section class="headline">
+            <div class="text-wrapper">
+                <h1>Felipe<br>Oyarzo</h1>
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of
+                    my works. If you like what you see and have a project you need coded, don't hesitate and contact me.
+                </p>
+            </div>
+            <button class="button" type="button" id="btnHeadline">Start collaboration</button>
+            <div class="geometric"></div>
+            <!-- <img src="./assets/img/image_geometry_2.png" alt="Geometric shapes" class="geometric"> -->
```
Comment (danmainah) [inline index.html]: - [x] [optional] Kindly ensure you don't have commented code in your codebase, its not good practise

**Review by danmainah — APPROVED** [review]
Hi @jfoyarzo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (jfoyarzo) [reply index.html]: Thanks for your feedback @danmainah , I totally forgot to delete that commented code, I appreciate it :)

**Pull request merged**